### PR TITLE
Review edit

### PR DESCRIPTION
**Pull request by Mikechavo**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -104,6 +104,9 @@ const App = () => {
       .catch((error) => console.log("logout errors: ", error))
   }
 
+  const updateReview = (editReview, id) => {
+
```
Comment (elysemontano) [inline src/App.js]: While I usually would say to remove any console logs, until your fetch calls are implemented, I would suggest having a console log as a place holder showing the content being passed in as params.

File: src/pages/BeerShow.js [hunk 2]
```diff
@@ -30,7 +31,14 @@ const BeerShow = () => {
       <Review reviews={beerReviews} />
       </>
     )}
+    <NavLink to={`/reviewedit/${currentReview.id}
+          `} className="nav-link">
+          <Button>
+            Edit Review
+          </Button>
+        </NavLink>
```
Comment (elysemontano) [inline src/pages/BeerShow.js]: Can anyone see this button or only logged-in users?

File: src/pages/ReviewEdit.js [hunk 3]
```diff
@@ -1,8 +1,106 @@
...
+    }
+  }, [currentReview])
+
+  const [editReview, setEditReview] = useState({
+    city: '',
+    state: '',
+    review_text: '',
+    rating: 0,
+    user_id: 0,
+    beer_id: 0,
```
Comment (elysemontano) [inline src/pages/ReviewEdit.js]: Foreign keys should not be hardcoded.

File: README.md [hunk 4]
```diff
@@ -68,3 +68,12 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+
+ 
+   const handleSubmit = () => {
+    // updateReview(editReview, currentReview.id)
+    // navigate(`/reviews/${currentReview.id}`)
+    
+  }
+ <Button onClick={handleSubmit}>Save</Button>
```
Comment (elysemontano) [inline README.md]: Do you need this in the README?

**Review by elysemontano — CHANGES_REQUESTED** [review]
Please see comments in README, App.js, BeerShow, and ReviewEdit

**Review by Mikechavo — COMMENTED** [review]
Changes made

File: src/pages/ReviewEdit.js [hunk 5]
```diff
@@ -1,9 +1,107 @@
...
+    }
+  }, [currentReview])
+
+  const [editReview, setEditReview] = useState({
+    city: '',
+    state: '',
+    review_text: '',
+    rating: 0,
+    user_id: current_user.id,
+    beer_id: currentReview ? currentReview.beer_id : 0,
```
Comment (elysemontano) [inline src/pages/ReviewEdit.js:31]: Is this association optional?  If so, this makes sense, if not, hardcoding 0 as the else will likely cause problems in the future.

**Review by elysemontano — APPROVED** [review]
Approving, but left an additional comment in ReviewEdit to look at and consider before merging

**Pull request merged**